.changeset/chilled-pugs-promise-2.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': patch
+---
+
+Adjusted `Dialog` and `ExpandableBlock` to round some CSS `transform` values. This helps avoid blurry text on Windows in some cases.
```

Comment:
nit/optional: If this PR also led (or could lead) to improvements in the `InformationPanel` too, should it also be mentioned in another changeset (since it's unrelated to the rounded transform change)?

Reply:
I don't think it's worth mentioning, it's a "just in case" type of thing.